.bumpversion.cfg

```diff
@@ -0,0 +1,8 @@
+[bumpversion]
+current_version = 1.0.0
+commit = True
+tag = True
+
+[bumpversion:file:setup.py]
+search = __version__ = {current_version}
+replace = __version__ = {new_version}
```

Comment:
Don't forget to include additional files in which the version number is included:
* ``sonar-project.properties``